File: 001/001.c
#include <stdio.h>

int main(void)
{
    int i = 0, sum = 0;

    for (i = 0; i < 1000; ++i) {
        if ((i % 3 == 0) || (i % 5 == 0)) {
            sum += i;
        }
    }

    printf("Result: %d\n", sum);
    return 0;
}

File: 001/001.cs
using System;
using System.Linq;

public class Problem001
{
    public static void Main()
    {
        var result = Enumerable.Range(0,1000)
            .Where(n => n % 3 == 0 or n % 5 == 0).Sum();
        Console.WriteLine("Result {0}", result);
    }
}
